server/views/pages/githubRepoRequestForm.njk

```diff
@@ -7,40 +7,56 @@
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 
 {% block content %}
+  <div class="govuk-grid-row govuk-body">
+    {% include 'partials/errorSummary.njk' %}
 <h1 class="govuk-heading-xl">Github repository Request Form</h1>
 
 <h2>After submitting this form, SRE team will run the bootstrap job to create the github repo and service catalog entries. </h2>
 
 <form action="/forms/github-repo-request-form" method="POST">
+    {%- set teams = [] -%}
     {%- set products = [] -%}
     {%- for product in productList -%}
     {%- set products = (products.push({
```

Comment:
Not going to hold back on this one as it's personal preference, but doing logic and loops like this to build a data structure is best done in the routes file and passed in. Makes the njk file cleaner as well 🙂